### PR TITLE
generate annotation docs

### PR DESCRIPTION
**Pull request by estute**
This makes a call to `code_annotations` to create the annotation docs for each IDA, and bundle them into their own sub-directory.

### CODE REVIEW

File: scripts/feature_toggle_report_generator.py [hunk 1]
```diff
@@ -314,6 +315,36 @@ def link_annotation_reports_to_idas(idas, annotation_report_files_path):
...
+    os.mkdir(output_path)
+    for ida_name, ida in idas.items():
+        if not ida.annotation_report_path:
+            continue
+        temp_annotation_report = os.path.join(
+            'reports', os.path.basename(ida.annotation_report_path)
+        )
+        shutil.copyfile(ida.annotation_report_path, temp_annotation_report)
+        annotation_config = AnnotationConfig('feature_annotations.yml', 0)
+        report_files = (io.open(temp_annotation_report, 'r'), )
```
Comment (jmbowman) [inline scripts/feature_toggle_report_generator.py]: Probably a little cleaner to put this code in a `with io.open(temp_annotation_report, ...` context manager, to make sure the file is closed at the right time.

**Comment by estute** [comment]
@jmbowman fixed up the issue you mentioned, and got it running tests with travis. Since this repo doesn't contain any Django code yet, I ripped out some of the references in tox.ini/travis.yml. Additionally, once I tested w/ production data, it exposed some flaws in the code, which have been fixed.

File: tox.ini [hunk 2]
```diff
@@ -29,15 +29,10 @@ ignore = D101,D200,D203,D212,D215,D404,D405,D406,D407,D408,D409,D410,D411,D412,D
 match-dir = (?!migrations)
 
 [pytest]
-DJANGO_SETTINGS_MODULE = test_settings
-addopts = --cov edx_toggles --cov-report term-missing --cov-report xml
```
Comment (jmbowman) [inline tox.ini]: Not measuring code coverage?

Comment (estute) [reply tox.ini]: fair point. fixed.

**Pull request merged**